ycmd.el

```diff
@@ -701,6 +721,42 @@ Returns the new value of `ycmd-force-semantic-completion'."
       (setq it (list it)))
     (mapcar 'expand-file-name it)))
 
+(defvar ycmd--mode-keywords-loaded nil)
+
+(defun ycmd--get-keywords (buffer)
+  "Get syntax keywords for BUFFER."
+  (with-current-buffer buffer
+    (let ((mode major-mode))
+      (unless (memq mode ycmd--mode-keywords-loaded)
```

Comment:
Won't this function return nil every time it's called after the first time for a mode? We populate ycmd--mode-keywords-loaded the first time, so subsequent calls will do nothing. What am I missing?